juriscraper/opinions/united_states/state/iowa.py

```diff
@@ -124,3 +125,47 @@ def extract_next_page_url(self, html):
         path = '//div[contains(./@class, "pagination-next-page")]//a/@href'
         elements = html.xpath(path)
         return elements[0] if elements else False
+
+    def extract_from_text(self, scraped_text: str) -> dict:
+        """Extract lower court from the scraped text.
+
+        :param scraped_text: The text to extract from.
+        :return: A dictionary with the metadata.
+        """
+        pattern = re.compile(
+            r"""
+            (?:
+               Appeals?\s+from\s+the\s+
+              | On\s+review\s+of\s+the\s+report\s+of\s+the\s+
+              | On\s+application\s+of\s+the\s+
+            )
+            (?P<lower_court>[^,.]+?)
+            (?:,\s*
+                (?P<lower_court_judge>[^,]+?)
+                ,\s*judge
+            )?
+            \.
+            """,
+            re.X | re.IGNORECASE,
+        )
+
+        result = {}
+        if match := pattern.search(scraped_text):
+            lower_court = re.sub(
+                r"\s+", " ", match.group("lower_court")
+            ).strip()
+
+            result["Docket"] = {
+                "appeal_from_str": lower_court,
```

Comment:
Failing in this opinion, the lower court has some extra words:

```
case_dates: "2025-05-16"
case_names: "Iowa Supreme Court Attorney Disciplinary Board v. Karmen R. Anderson"
download_urls: "https://www.iowacourts.gov/courtcases/23628/embed/SupremeCourtOpinion"
appeal_from_str: "report of the Iowa Supreme Court Grievance Commission"
```

Lower court judge looks good.